# Internal fix for mishandling of invalid Unicode character sequences.
# See bug 21021316.  Not for upstream.
--- tests/intl.right	Fri Nov  6 02:48:31 2015
+++ tests/intl.right	Fri Nov  6 03:30:58 2015
@@ -46,6 +46,6 @@
 0000003
 0000000   101 040 302 243 040 305 222 012                                
 0000010
-./unicode3.sub: line 2: 5§@3™+ÆS8Ÿ¢ê³: command not found
-5§@3™+ÆS8Ÿ¢ê³
+./unicode3.sub: line 2: $'5\247@3\231+\306S8\237\242\352\263': command not found
+$'5\247@3\231+\306S8\237\242\352\263'
 + : $'5\247@3\231+\306S8\237\242\352\263'
